static/app.css

```diff
@@ -52,11 +52,17 @@ h1 {
   line-height: 120%;
 }
 
-h3 {
-  text-transform: uppercase;
+h2{
   font-size: 1.875rem;
   color: #d36767;
   margin: 0.5rem 0;
+  font-weight: 600;
+}
+
+h3 {
```

Comment:
Did you check if this affects other pages?

Reply:
yes, they all overwrite it with tailwind classes. Also think some styling for H tags is not a bad thing.